string control_type
---
bool status